src/ui_fsmenu/keyboard_options.cc

```diff
@@ -187,15 +266,29 @@ KeyboardOptions::KeyboardOptions(Panel& parent)
 	};
 	create_tab(_("General"), KeyboardShortcut::kCommon__Begin, KeyboardShortcut::kCommon__End);
 	create_tab(_("Main Menu"), KeyboardShortcut::kMainMenu__Begin, KeyboardShortcut::kMainMenu__End);
-	create_tab(_("Game"), KeyboardShortcut::kInGame__Begin, KeyboardShortcut::kInGame__End);
 	create_tab(_("Editor"), KeyboardShortcut::kEditor__Begin, KeyboardShortcut::kEditor__End);
+	create_tab(_("Game"), KeyboardShortcut::kInGame__Begin, KeyboardShortcut::kInGame__End);
+	create_tab(
+	   _("Fastplace"), KeyboardShortcut::kFastplace__Begin, KeyboardShortcut::kFastplace__End);
 
 	buttons_box_.add_inf_space();
 	buttons_box_.add(&reset_, UI::Box::Resizing::kAlign, UI::Align::kCenter);
 	buttons_box_.add_inf_space();
 	buttons_box_.add(&ok_, UI::Box::Resizing::kAlign, UI::Align::kCenter);
 	buttons_box_.add_inf_space();
 
+	tabs_.sigclicked.connect([this, all_keyboard_buttons, generate_title]() {
+		if (tabs_.active() == 4 && game_.get() == nullptr) {
```

Comment:
uhh, better not changing tab order from now on, right? :D maybe make this more visible by defining a constant and setting Fastplace tab explicitly to that index (if this is possible)

Reply:
added a variable that detects and stores the tab's index